Tests/Singulink.Globalization.Currency.Tests/MoneyTests/ToStringTests.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Shouldly;
+
+namespace Singulink.Globalization.Tests.MoneyTests;
+
+[TestClass]
+public class ToStringTests
+{
+    private static readonly CultureInfo _enUS = CultureInfo.GetCultureInfo("en-US");
+
+    [TestMethod]
+    public void ToString_DefaultFormat()
+    {
+        new Money(100m, "USD").ToString(null, null).ShouldBe("USD\u00A0100.00");

```

Comment:
This should set the current culture before you call ToString, i.e. `CultureInfo.CurrentCulture = _enUS;`, otherwise it will fail the test on machines that use some different cultures.